src/scss/index.scss

```diff
@@ -5,3 +5,4 @@
 @import "components/index";
 @import "helpers/index";
 @import "overrides";
+/* END: SCSS-generated Styles */
```

Comment:
This is for debugging purposes and will be removed at the end of the SCSS conversion.